cda-core/src/diag_kernel/diagservices.rs

```diff
@@ -211,7 +210,8 @@ impl DiagServiceResponseStruct {
                 errors,
             });
         }
-        let start = Instant::now();
+
+        let start = std::time::Instant::now();
```

Comment:
question: is logging the time actually necessary in combination with the tracing::instrument?